tests/rpminspect/rpminspect.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+	rlPhaseStartSetup
+		# TODO: Add interface to get necessary rpminspect.yaml
+		# fetch-my-conf.py
+		rlRun "freshclam" 0 "Update clam antivirus database"
+		rlRun "tmp=\$(mktemp -d)" 0 "Create tmp directory"
+		rlRun "pushd $tmp"
+		rlRun "set -o pipefail"
+		rlRun "mkdir -p ./inspect_builds/$PACKIT_PACKAGE_NVR" 0 "Make basic directory layout"
+	rlPhaseEnd
+
+	rlPhaseStartTest
+		# Get OS info of the current distro. Maybe there is a more modern approach?
+		source /etc/os-release
+		# Get the most recent koji build
+		if [[ -n "${RPMINSPECT_KOJI_BUILD}" ]]; then
+				rlFail "Not implemented for koji tags"
+		else
+			if rlIsFedora; then
+				# TODO: Allow for other variables defining PACKAGE_NAME
+				rlRun -s "/usr/bin/koji list-tagged --latest --inherit --quiet f${VERSION_ID} ${PACKIT_PACKAGE_NAME}" 0 "Get latest koji build"
```

Comment:
We can maybe add a `README.md` file into each plan directory and define the env. vars?

Reply:
With this, we can actually include this into `packit.dev`.

Reply:
If it's navigable via GitHub UI, it can easily be included in packit.dev page similarly to [research ](https://github.com/packit/research/) and [deployment](https://github.com/packit/deployment/).

Reply:
Just a directory structure with markdown files and it will work.